content/en/metrics/guide/metrics-without-limits-getting-started.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Getting Started with Metrics without Limits™
+kind: guide
+is_beta: true
+---
+
+<div class="alert alert-warning">This functionality is in private beta.</div>
+
+## Overview
+
+This guide covers how to get started with Metrics without Limits™ and quickly gain the most value from the feature. 
+
+Review the [Metrics without Limits™ FAQ][1].
+
+
+1. Begin by configuring your Top 20 metrics on the Usage page.
+   You can [configure metrics][2] both in the Metrics Summary UI and with the API.
```

Comment:
I don't know Metrics very well, but I feel like the bulk of the instructions in this getting started guide are right here in these two lines. It tells me that i should and can do these things, but it's not super clear on how, and the link points to a fairly generic page. How do I find out how to do these things?

Reply:
Fair - we can include the image already included on the Distribution Metrics' doc / wording from the Customize Tagging section here: https://docs.datadoghq.com/metrics/distributions/#customize-tagging

Reply:
Woops, I messed up this link, fixed

Reply:
I think this also sort of ties in with our OKR for the getting started with metrics page, once that's completed we can add links to it from here